DOCS/license-matching-guidelines-and-templates.md

```diff
@@ -203,31 +203,31 @@ To implement this guideline, use the `<optional>` XML element tag as described i
 
 ### B.14.1 Purpose <a name="B.14.1"></a>
 
-To avoid a license mismatch due to a difference in a hyperlink protocol (e.g. http vs. https).
+To avoid a license mismatch due to a difference in a hyperlink protocol (e.g. HTTP vs. HTTPS).
 
 ### B.14.2 Guideline <a name="B.14.2"></a>
 
-HTTP:// and HTTPS:// should be considered equivalent. 
+"http://" and "https://" should be considered equivalent. 
```

Comment:
Why? It does not match the change above.

Reply:
Strings "http://" and "https://" are the first part of a URL.
"http" and "https" in this context are URI schemes of URL.

`http` is an URI scheme for Hypertext Transfer Protocol (HTTP).

`https` is an URI scheme for Hypertext Transfer Protocol Secure (HTTPS).

HTTP and HTTPS are the name of the protocols, they are written in uppercase (as an initialism).
But their corresponding URI schemes, http and https, are written in lowercase, [similar to other URI schema](https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml).

HTTP and HTTPS URI schemes are defined here:
https://www.rfc-editor.org/rfc/rfc9110.html#name-http-related-uri-schemes

If the guideline is about a comparison of URL strings (as it shows in "xxxx://"), we should use a convention for URL.

Reply:
@bact - is there a reason you changed the capitalization in two places here?